QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+$QUEST_JOBSTEP_20150317_000777	"The game doesn't end before the 3rd round, so I think it's pointless to talk about 'compensation' without the game ending…"
+$QUEST_JOBSTEP_20150317_000778	Recognizing that he has already won two rounds and had no reason to refuse the 3rd round, and for the reason that the game is certainly not over yet, Rumpelstiltskin abided to Lydia Schaffen's words. And so the final round begins, Lydia Schaffen measures her last arrow, and called out as she pulled the arrow back. "The target is… the moon in the sky!" Rumpelstiltskin got on the arrow and into a horseback riding posture as soon as the arrow left the bowstring, and not before long, he realized what he heard. 
+$QUEST_JOBSTEP_20150317_000779	And as he rode the arrow towards the moon, he also realized the fact that the game cannot end. As time passes, the arrow's power subsides and begin to fall towards the ground, however, Lydia Schaffen shot another arrow and caused the arrow that Rumpelstiltskin rode on to stop still, floating in the air.
+$QUEST_JOBSTEP_20150317_000780	Rumpelstiltskin thought he'd ask Lydia Schaffen what should he do to get out of his situation, but before he could, Lydia Schaffen began to talk, "For the game to end, you need to ride the arrow until it arrive at its target. The game cannot end before then. After you have touched the moon, and if I come back, then I will definitely accept the compensation."
+$QUEST_JOBSTEP_20150317_000781	According to the anecdote, Rumpelstiltskin has not reached the moon yet, and is still flying there with the magic in his body. It could also be the case that he returned before his demonic life span ended.
+$QUEST_JOBSTEP_20150317_000782	Nevertheless, as we know, Lydia Schaffen has passed away long ago. So while the demon did not lose the game, he has lost his chance to win the game forever. Still, the one matter left that we, as the current generation, are curious about is what the agreed compensation between Lydia Schaffen and Rumpelstiltskin was. Of course, we could ask a particular person, err, there is a thing that may still exist. However, as we do not know whether he has returned or not, it will be hard to settle this lingering matter for quite some time.
+$QUEST_JOBSTEP_20150317_000783	Atman Extension
+$QUEST_JOBSTEP_20150317_000784	Atman Extension
+$QUEST_JOBSTEP_20150317_000785	Congratulations on receiving this book, which no criminal may see. There are many ways to use magic, but the magic used by Linkers are a particular case. Other mages exercise effect-based powers rather than magic. That is to say, to fight an enemy with a fireball, or with lightning, does not truly bind the enemy with magic. 
+$QUEST_JOBSTEP_20150317_000786	These attacks only use magic to make a fireball and lightning. Then, the mages controls the magical energy's direction and destination, releasing it after settling on both.
```

Comment:
"mages controls" needs to be changed to either "mages control" or "mage controls."